### PR TITLE
RN-01 login component

### PR DESCRIPTION
**Pull request by ghost**
finish the login options component.  This is where users have the option to go in signup with their email, google, or Facebook.

### CODE REVIEW

**Comment by ghost** [comment]
Screen Shot 2020-11-12 at 9 09 22 PM

File: app/components/root.jsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import { NativeRouter } from 'react-router-native';
+// import { NativeRouter } from 'react-router-native';
 import App from "./app";
 import { Provider } from 'react-redux';
 
 const Root = ({store}) => {
   return(
     <Provider store={store}>
-        <NativeRouter>
+        {/* <NativeRouter> */}
```
Comment (samwalker191) [inline app/components/root.jsx:9]: Was this still not working for you?

File: app/components/session/login_options_component.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return(
+    <View style={[layoutStyles.backgroundLayout,loginOptionsStyles.loginLayoutCentering]}>
+      <View style={loginOptionsStyles.contentContainer}>
+        <View style={loginOptionsStyles.circleElementTitleContainer}> 
+          <View style={loginOptionsStyles.circleElement}>
+
+          </View>
+
+          <View style={loginOptionsStyles.Title}>
+              BGB
```
Comment (samwalker191) [inline app/components/session/login_options_component.jsx:16]: When trying to run this on a phone, I get the error saying that these need to be wrapped in a `<Text></Text>` component. I don't think React Native lets us have free floating text.

Comment (samwalker191) [reply app/components/session/login_options_component.jsx:16]: This applies to all the 'floating' text.

File: app/styles/styles.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    backgroundColor: "#C4C4C4",
+  },
+
+  Title: {
+   width: 174,
+   height: 84,
+   justifyContent: "center",
+   alignItems: "center",
+   fontSize: 72,
+   fontStyle: "Italic",
```
Comment (samwalker191) [inline app/styles/styles.js]: I was getting an error with the capitalized 'Italic' here. Says it needs to be lowercase.

File: app/styles/styles.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    backgroundColor: "#4C55AB",
+    color: "#FFFFFF"
+  },
+
+  redirectLink: {
+    width: 245,
+    height: 28,
+    fontWeight: 100,
+    fontSize: 21,
+    fontStyle: "Italic",
```
Comment (samwalker191) [inline app/styles/styles.js]: This should also be lowercase.

File: app/styles/styles.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+  sessionFacebook: {
+    backgroundColor: "#4C55AB",
+    color: "#FFFFFF"
+  },
+
+  redirectLink: {
+    width: 245,
+    height: 28,
+    fontWeight: 100,
```
Comment (samwalker191) [inline app/styles/styles.js]: Got an error saying this should be a string instead of a number.

File: app/styles/styles.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+  Title: {
+   width: 174,
+   height: 84,
+   justifyContent: "center",
+   alignItems: "center",
+   fontSize: 72,
+   fontStyle: "Italic",
+   color: "#FFFFFF",
+   fontWeight: "Bold"
```
Comment (samwalker191) [inline app/styles/styles.js]: Same error as above, I think this should be lowercase.

**Review by samwalker191 — CHANGES_REQUESTED** [review]

**Comment by samwalker191** [comment]
Additionally, I think we need to specify `fontFamily`. It seems that the default font is different depending on the device, with Android being 'Roboto' and ios being 'San Francisco' i think. 

We can actually create components that we can use instead of Text components to always apply the font style. So we dont have to write it in the style prop a bunch of times.

https://reactnative.dev/docs/text#limited-style-inheritance
https://medium.com/@KentaKodashima/react-native-create-a-custom-text-component-with-custom-font-family-61c5fcdf9388

**Comment by ghost** [comment]
@samwalker191 Ok so i made all the fixes so please let me know if those work on your android.  It seems like they worked on my I phone.  I also starting a text component template but didn't integrate it.  I was thinking that we make another issue for that and I can work on getting that made after we merge this PR.

**Review by samwalker191 — APPROVED** [review]

**Comment by samwalker191** [comment]
Works great on my phone! Looks awesome too. And yea, i think the custom text component can be its on PR.

Looks dope to me.

**Comment by ghost** [comment]
alright ima go ahead and squash my commits and then merge

**Comment by ghost** [comment]
Thank you for checking so quickly !

**Pull request merged**